File: app/src/main/java/com/yorisprayogo/mvptemplate/di/PerActivity.java
package com.yorisprayogo.mvptemplate.di;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Scope;

/**
 * Created by janisharali on 27/01/17.
 */

@Scope
@Retention(RetentionPolicy.RUNTIME)
public @interface PerActivity {
}

File: app/src/main/java/com/yorisprayogo/mvptemplate/di/PerActivity.kt
package com.yorisprayogo.mvptemplate.di

import javax.inject.Qualifier
import javax.inject.Scope

/**
 * Created by janisharali on 27/01/17.
 */

@Scope
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
annotation class PerActivity

